sisyphus/job.py

```diff
@@ -516,10 +516,12 @@ def _sis_cleanup(self):
                 try:
                     if not gs.JOB_CLEANUP_KEEP_WORK:
                         shutil.rmtree(os.path.abspath(self._sis_path(gs.JOB_WORK_DIR)))
+                    if not gs.JOB_CLEANUP_KEEP_INPUT:
+                        shutil.rmtree(os.path.abspath(self._sis_path(gs.JOB_INPUT)))
                     files = [
                         i
                         for i in os.listdir(self._sis_path())
-                        if i not in (gs.JOB_OUTPUT, gs.JOB_INFO, gs.JOB_WORK_DIR)
+                        if i not in (gs.JOB_OUTPUT, gs.JOB_INFO, gs.JOB_WORK_DIR, gs.JOB_INPUT)
```

Comment:
If we don't delete the input of course zip it here if its not going to be kept outside.